nullaway/src/test/java/com/uber/nullaway/NullAwayAutoSuggestNoCastTest.java

```diff
@@ -55,12 +55,41 @@ public void setup() {
     flagsNoAutoFixSuppressionComment = b.build();
   }
 
+  // In EP 2.6.0 the newInstance() method we use below is deprecated.  We cannot currently address
+  // the warning since the replacement method was only added in EP 2.5.1, and we still want to
+  // support EP 2.4.0.  So, we suppress the warning for now
```

Comment:
If it ever gets removed and we want to still support say EP `2.4.0` and `2.7.0`, we could do some reflection trickery here, but it's probably cleaner for now to use the deprecated method :)